packages/data-portal-explore/src/components/FileTable.tsx

```diff
@@ -918,10 +930,17 @@ export class FileTable extends React.Component<IFileTableProps> {
         super(props);
         makeObservable(this);
 
-        this.columnVisibility = _.mapValues(
-            _.keyBy(this.defaultColumns, (c) => c.name),
-            () => true
+        const columnVisibilityMap = _.mapValues(
+            _.keyBy(
+                this.defaultColumns,
+                (c) => (c as IEnhancedDataTableColumn<Entity>).name
+            ),
+            (c) =>
+                typeof c !== 'number' &&
```

Comment:
I think we can remove this `typeof` check
```suggestion

```

Reply:
When i remove the `typeof` check it gives the following error:

```
Property 'omit' does not exist on type 'number | IEnhancedDataTableColumn<Entity> | ((...items: IEnhancedDataTableColumn<Entity>[]) => number) | ... 31 more ... | (<A, D extends number = 1>(this: A, depth?: D | undefined) => FlatArray<...>[])'.
  Property 'omit' does not exist on type 'number'.
```

Reply:
Hmm, interesting. Do you see this error in the console or is this an error generated by the IDE? Sometimes IDEs complain about the code but the actual build is successful. I don't get any errors when I build it with `yarn build`, no errors with `yarn dev` either? Which typescript version are you using?

Reply:
You can also try setting the parameter type instead of type casting. Below code also works for me locally.

```JavaScript
const columnVisibilityMap = _.mapValues(
    _.keyBy(
        this.defaultColumns,
        (c: IEnhancedDataTableColumn<Entity>) => c.name
    ),
    (c: IEnhancedDataTableColumn<Entity>) => !c.omit
);
```

Reply:
@onursumer I got this error on my IDE. Currently I don't see it anymore, a bit strange, but I'm pushing the code.